e2e-tests/tests/shift-rearrange-elements.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { expect } from "@playwright/test";
+import { studioTest } from "./infra/studioTest.js";
+import fs from "fs";
+
+const expectedPage = fs.readFileSync(
+  "./tests/__fixtures__/shift-rearrange-elements-expected-page.tsx",
+  "utf-8"
+);
+
+studioTest(
+  "can rearrange multiple elements in tree using shift",
```

Comment:
can this test be updated to have a more visible change? right now, the preview panel doesn't show any change from the rearranging